.github/workflows/e2e.maven.workflow_dispatch.main.default.slsa3.yml

```diff
@@ -0,0 +1,49 @@
+name: Maven e2e test - simple
+on:
+  schedule:
+    - cron: "0 6 * * *"
+  workflow_dispatch:
+
+permissions: read-all
+
+env:
+  GH_TOKEN: ${{ github.token }}
```

Comment:
nit: We may want to create a PAT token for this.

/cc @laurentsimon

Reply:
yes, I think it's also necessary to push or create releases. I recall some workflows were not triggered unless it used a PAT. Remind me: what permissions do we need for the PAT? Is it documented somewhere?

Reply:
Is this a TODO on my side_

Reply:
no. Use `GH_TOKEN: ${{ secrets.E2E_GO_TOKEN }}` for the time being. I created https://github.com/slsa-framework/example-package/issues/263 for tracking

Reply:
not yet updated in code